.github/CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ to you, it likely is not to the person asking the question.
 
 ## Becoming a Team Member
 
-The PRAW's team is always iterested in expanding PRAW's active team member base
+The PRAW's team is always interested in expanding PRAW's active team member base
```

Comment:
While we're here, it this should either read `The PRAW team` or `PRAW's team`. Care to make that fix as well?

Thanks!

Reply:
Done!